job.go

```diff
@@ -143,6 +143,14 @@ func (c *connection) connect(iv time.Duration) error {
 		return nil
 	}
 	dsn := c.url.String()
+
+	//as sqlx doesn't allow escaped passwords,
+	//we replace the escaped one by the unescaped password
+	if passwordUnescaped, passwordIsSet := c.url.User.Password(); passwordIsSet == true {
```

Comment:
The second argument `passwordIsSet` is usually named `ok` or `found`. Would be more concise and common.